Code/ForceField/Wrap/ForceField.cpp

```diff
@@ -130,6 +131,64 @@ PyObject *ForceFieldGetExtraPointLoc(PyForceField *self, unsigned int idx) {
   return res;
 }
 
+double PyForceField::calcEnergyWithPos(const python::object &pos) {
+  PRECONDITION(this->field, "no force field");
+  if (pos != python::object()) {
+    const python::list &posList = reinterpret_cast<const python::list &>(pos);
```

Comment:
reinterpret_cast is ordinarily a bad idea.  What happens if we pass in something that isn't a python::list?

https://stackoverflow.com/questions/573294/when-to-use-reinterpret-cast

I suggest simply having
```
double PyForceField::calcEnergyWithPos(const python::list &pos) {
```

This will enforce the proper type checking when boost wraps the function.

Reply:
@bp-kelley Good point. There are other places in the RDKit code where a similar hope-for-the-best paradigm is used. I'll submit a PR later.